### PR TITLE
#162948794 set an empty react app

### PR DESCRIPTION
**Pull request by Georgeygigz**
**What does this PR do?**
- It creates a new react application

**Description of Task to be completed?**
- Creating a new application using `npm create-react-app .`

**How should this be manually tested?**
- clone this repo `https://github.com/andela/ah-django-unchained-frontend`
- `cd ah-django-unchained-frontend`
- Install the required dependencies `npm install`
- `npm start to run the local server`
- The following message show that your server has been started successfully
 ``` 
You can now view ah-django-unchained-frontend in the browser.
  Local:            http://localhost:3004/
  On Your Network:  http://10.6.54.151:3004/

Note that the development build is not optimized.
To create a production build, use npm run build. 
```
**Pivotal Tracker story**

[#162948794](https://www.pivotaltracker.com/story/show/162948794)

### CODE REVIEW

**Comment by cosmas28** [comment]
@Georgeygigz kindly add linting using Airbnb eslint style guide as indicated in the chore description. Linting provides an extra measure of code quality by analyzing code and highlighting potential errors. It can also ensure consistency across projects with multiple developers.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "name": "ah-django-unchained-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "eslint-config-airbnb": "^17.1.0",
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
+    "react-scripts": "2.1.3"
+  },
+  "scripts": {
```
Comment (cosmas28) [inline package.json:11]: Kindly add appropriate linting scripts.

**Review by cosmas28 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Georgeygigz) [reply package.json:11]: Okay noted

**Comment by cosmas28** [comment]
@Georgeygigz Good job for adding linters. However, I requested earlier that you should add scripts for running ESlint against the code directory and  Fixing eslint errors automatically.

**Comment by cosmas28** [comment]
Good improvement @Georgeygigz , one more thing before I merge this PR. Please run ESlint against our code directory and fix the eslint errors. e.g change the `App.js` to `App.jsx` because we are using JXS.

**Review by cosmas28 — APPROVED** [review]

**Pull request merged**